java/src/jmri/jmrix/dccpp/DCCppInitializationManager.java

```diff
@@ -27,12 +27,18 @@ protected void init() {
             log.debug("Init called");
         }
 
-	
-	String base_station = systemMemo.getDCCppTrafficController().getCommandStation().getBaseStationType();
-	String code_build = systemMemo.getDCCppTrafficController().getCommandStation().getCodeBuildDate();
+	String base_station = "Unknown";
+	String code_build = "Unknown";
+
+	if (systemMemo.getDCCppTrafficController().getCommandStation() != null) {
+	    base_station = systemMemo.getDCCppTrafficController().getCommandStation().getBaseStationType();
+	}
+	if (systemMemo.getDCCppTrafficController().getCommandStation() != null) {
+	    code_build = systemMemo.getDCCppTrafficController().getCommandStation().getCodeBuildDate();
+	}
 	/* First, we load things that should work on all systems */
 	if (systemMemo.getPowerManager() == null)
-	    log.error("Power Manager not created!");
+	    log.error("Power Manager not (yet) created!");
```

Comment:
The style guidelines we follow (and have configurations for NetBeans and Eclipse to automatically apply) require braces (```{}```) for all if statements, including single line statements.